packages/venia-ui/lib/components/ToastContainer/toast.css

```diff
@@ -48,7 +49,7 @@
 }
 
 .infoToast > .icon {
-    color: info;
+    --stroke: info;
```

Comment:
I think this broke when the Icon component was re-factored, or I broke it. Either way, this should work as expected now.